### PR TITLE
Add instructions to Test SEP-8 Payments

### LINKED ISSUE #160: Update Readme.md with instructions to test the SEP-8 flow

### PR DESCRIPTION
**Pull request by marcelosalloum**
### What

Add instructions to Test SEP-8 Payments in the Demo Wallet using our SEP-8 reference server sep8-server.dev.stellar.org.

Closes #160

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,22 @@ feel free to clone or copy any pieces that may be helpful.
...
+You can use the demo wallet to test Regulated Assets ([SEP-8]), Hosted Deposit and Withdrawal ([SEP-24]) and Cross-Border Payments ([SEP-31]) with any home domain that has a Stellar Info File (also known as [SEP-1], or a stellar.toml file). The instructions below are for demo-ing standard integrations supported by Stellar test server, testanchor.stellar.org or by the [SEP-8] reference server, sep8-server.dev.stellar.org. For these integrations, the logs to the right of the screen will show every network call.
+
+### Demo-ing a Regulated Asset Payment ([SEP-8])
 
-### Demo-ing a Deposit on Testnet with Hosted Deposit and Withdrawal (SEP-24)
+1. Click "Generate keypair", and then click "Create account" - this will create a balance of 10,000 XLM.
+2. Click “Add asset” and add `MYASSET` with the anchor home domain `sep8-server.dev.stellar.org`.
+3. Click “Add trustline” - this will allow you to hold MYASSET.
+4. Click in the "Copy" link at the right of your public key and use that value to get some unities of MYASSET using the link `https://sep8-server.dev.stellar.org/friendbot?addr=<paste_your_address_here>`.
```
Comment (quietbits) [inline README.md:26]:
```suggestion
4. Click on the "Copy" link on the right of your public key and use that value to get some unities of MYASSET using the link `https://sep8-server.dev.stellar.org/friendbot?addr=<paste_your_address_here>`.
```

File: README.md [hunk 2]
```diff
@@ -16,9 +16,22 @@ feel free to clone or copy any pieces that may be helpful.
...
+5. Select "SEP-8 Send" from the dropdown for MYASSET and click "Start" in the modal.
```
Comment (quietbits) [inline README.md:27]:
```suggestion
5. Select "SEP-8 Send" from the dropdown of MYASSET and click "Start" in the modal.
```

File: README.md [hunk 3]
```diff
@@ -16,9 +16,22 @@ feel free to clone or copy any pieces that may be helpful.
...
+6. In the "destination" field, input an address that also has a trustline to MYASSET. You can use this destination address for it if you don't have one: `GBID36ML6VVNPIF6SQATQW5QIBREQAVEHQIUMWDLQCSVIYX2IJGK4KPP`.
```
Comment (quietbits) [inline README.md:28]: Will we remember to update this address when the testnet resets?

**Review by quietbits — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (marcelosalloum) [reply README.md:28]: Good point! I'll remove the reference to that address so we don't risk becoming outdated.

Comment (marcelosalloum) [reply README.md]: Removed the sentence about the existing address in https://github.com/stellar/stellar-demo-wallet/pull/170/commits/6bb496a8fc971419c445d8277efa57ca51cb3d1c.

File: README.md (hunk 2 above)
Comment (marcelosalloum) [reply README.md]: Updated the wording in https://github.com/stellar/stellar-demo-wallet/pull/170/commits/6bb496a8fc971419c445d8277efa57ca51cb3d1c.

**Pull request merged**